Kudu.Services/Diagnostics/LogStreamManager.cs

```diff
@@ -433,11 +435,6 @@ private void OnRenamed(object sender, RenamedEventArgs e)
 
         private void OnError(object sender, ErrorEventArgs e)
         {
-            using (_tracer.Step("FileSystemWatcher.OnError"))
-            {
-                _tracer.TraceError(e.GetException());
-            }
-
```

Comment:
For simplicity, just don't trace it.   Instead, send error to client straight (codes a few lines below).